src/pages/public_registration/LogPanel.tsx

```diff
@@ -41,6 +43,68 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
     cacheTime: 1_800_000, // 30 minutes
   });
 
+  const logs: LogList = [];
+
+  if (registration.publishedDate) {
+    const registrationPublished: LogItem = {
+      modifiedDate: registration.publishedDate,
+      description: t('registration.status.PUBLISHED_METADATA'),
+      type: 'PublishingRequest',
+    };
+    logs.push(registrationPublished);
+  }
+
+  if (registration.publishedDate && registration.publishedDate < registration.modifiedDate) {
+    const registrationLastModified: LogItem = {
+      modifiedDate: registration.modifiedDate,
+      description: t('common.last_modified'),
+      type: 'PublishingRequest',
+    };
+    logs.push(registrationLastModified);
+  }
+
+  tickets.forEach((ticket) => {
+    switch (ticket.type) {
+      case 'PublishingRequest': {
+        if (ticket.status === 'Completed' && (ticket as PublishingTicket).approvedFiles.length > 0) {
```

Comment:
noen ganger har jeg brukt å ta castingen over i en egen const, for å slippe å gjøre samme cast flere ganger, men det går også for det samme
```suggestion
        const publishingTicket = ticket as PublishingTicket;
        if (ticket.status === 'Completed' && publishingTicket.approvedFiles.length > 0) {
```